tensorflow/lite/micro/testing/test_bluepill_binary.sh

```diff
@@ -27,30 +27,29 @@
 declare -r ROOT_DIR=`pwd`
 declare -r TEST_TMPDIR=/tmp/test_bluepill_binary/
 declare -r MICRO_LOG_PATH=${TEST_TMPDIR}
-declare -r MICRO_LOG_FILENAME=${MICRO_LOG_PATH}/logs.txt
+declare -r MICRO_LOG_FILENAME=${MICRO_LOG_PATH}logs.txt
 mkdir -p ${MICRO_LOG_PATH}
 
-docker build -t renode_bluepill \
-  -f ${ROOT_DIR}/tensorflow/lite/micro/testing/Dockerfile.bluepill \
-  ${ROOT_DIR}/tensorflow/lite/micro/testing/
+if [ -e $MICRO_LOG_FILENAME ]; then
+    rm $MICRO_LOG_FILENAME &> /dev/null
+fi;
 
 exit_code=0
-# running in `if` to avoid setting +e
-if ! docker run \
-  --log-driver=none -a stdout -a stderr \
-  -v ${ROOT_DIR}:/workspace \
-  -v /tmp:/tmp \
-  -e BIN=/workspace/$1 \
-  -e SCRIPT=/workspace/tensorflow/lite/micro/testing/bluepill.resc \
-  -e EXPECTED="$2" \
-  -it renode_bluepill \
-  /bin/bash -c "/opt/renode/tests/test.sh /workspace/tensorflow/lite/micro/testing/bluepill.robot 2>&1 >${MICRO_LOG_FILENAME}"
+
+if ! BIN=${ROOT_DIR}/$1 \
+  SCRIPT=${ROOT_DIR}/tensorflow/lite/micro/testing/bluepill.resc \
+  LOGFILE=$MICRO_LOG_FILENAME \
+  EXPECTED="$2" \
+  ${ROOT_DIR}/tensorflow/lite/micro/tools/make/downloads/renode/test.sh \
+  ${ROOT_DIR}/tensorflow/lite/micro/testing/bluepill.robot \
+  -r $TEST_TMPDIR &> ${MICRO_LOG_PATH}robot_logs.txt
 then
   exit_code=1
 fi
 
 echo "LOGS:"
-cat ${MICRO_LOG_FILENAME}
+# Extract output from renode log
+cat ${MICRO_LOG_FILENAME} |grep 'uartSemihosting' |sed 's/^.*from start] *//g'
```

Comment:
We remove timestamps and log level info. It is also possible to output the whole log or create separete file for uart output and `cat` it here.